modules/apps/portal/portal-log4j/src/main/java/com/liferay/portal/log4j/internal/CompanyWebIdLogContext.java

```diff
@@ -35,6 +37,13 @@ public Map<String, String> getContext(String logName) {
 
 		if (companyId > 0) {
```

Comment:
@yuhai Please make it a shortcut when `companyId == 0`, so that it's clear the `StringPool.BLANK` has nothing to do with the value when `companyId > 0`.